docs/source/fundamentals/benefits.md

```diff
@@ -0,0 +1,133 @@
+---
+title: GraphQL benefits
+description: Why adopting GraphQL and Apollo will help you ship features faster
+---
+
+Managing data for modern applications is difficult. Developers have to aggregate data from multiple sources, distribute it upon multiple platforms, and plumb it into an app's UI. On top of that, front-end developers have to decide how to manage state on the client, all while executing complicated features such as caching and optimistic UI.
+
+Adopting GraphQL in your organization will ease these pain points considerably. Read on to learn how GraphQL's declarative approach to data fetching will simplify data transformation and speed up your API. You'll also learn how the Apollo platform enables faster development cycles thanks to its advanced ecosystem of tooling and excellent developer experience.
+
+<h2 id="declarative">Declarative data fetching</h2>
+
+One of the main advantages of adopting GraphQL is its declarative approach to data fetching. With GraphQL, there's no need to call multiple endpoints from the client or aggregate the data manually like you have to with traditional REST data fetching. Instead, you specify exactly the data you need and GraphQL gives you exactly what you asked for.
+
+With REST, you would have to call all of these endpoints for each item in the list, filter down the data you need, and aggregate all of the remaining data into the shape your components consume.
+
+```bash
+GET /api/dogs/breeds
+GET /api/dogs/images
+GET /api/dogs/activities
+```
+
+Not only is this approach time-consuming, it's also prone to error and difficult to reuse logic across platforms. Compare this with GraphQL's declarative way to query data:
+
+```graphql
+const GET_DOGS = gql`
+  query {
+    dogs {
+      id
+      breed
+      image {
+        url
+      }
+      activities {
+        name
+      }
+    }
+  }
+`;
+```
+
+Here, we're describing the shape of the object we want to receive from the server. GraphQL takes care of combining and filtering the data while returning exactly what we ask for.
+
+How do we use this query in our app? Apollo Client builds off of GraphQL's declarative approach to data fetching. In a React app, all of the logic for retrieving your data, tracking loading and error states, and updating your UI is encapsulated in a single Query component. This encapsulation makes composing your data fetching components with your presentational components a breeze! Let’s see how to fetch GraphQL data with Apollo Client in a React app:
```

Comment:
Let's put "Query" inside backticks/graves to indicate it's a code reference (to the below snippet), à la `` `Query` `` and yielding `Query`.